lti_consumer/models.py

```diff
@@ -526,3 +528,58 @@ def __str__(self):
 
     class Meta:
         app_label = 'lti_consumer'
+
+
+class CourseEditLTIFieldsEnabledFlag(ConfigurationModel):
```

Comment:
As we are moving this from edx-platform but we should name the model by convention and postfix with `Config` or `Configuration`. 
e.g. `CourseEditLTIFieldsEnabledConfig`

Reply:
I am not sure if we can change the model name here for backward compatibility. So having that insight would be very helpful.

Reply:
I think we can change the name, but I feel Flag is more appropriate.  There are a couple of other ConfigurationModel-based Models that use the Flag suffix (for instance PersistentGradesEnabledFlag). 

Given that this flag will now do more than prevent editing, perhaps the name change can be more drastic. For example:
`CourseAllowLTISharePIIConfig` or `CourseEnableLTISharingPIIFlag` or something similar.

Reply:
Sounds good to me - I am ok with a new pr for this change.